### PR TITLE
Adds Enginseer Robe

### PR DESCRIPTION
**Pull request by DerFlammenwerfer**
It's a variant of the normal techpriest robe designed to look better than the original, without removing the original for those who prefer it.

## About The Pull Request

Adds a variant of the Techpriest robes that looks more armored, and is an improvement in aesthetic over the original.

## Why It's Good For The Game

This was already approved months ago but never got added due to merge conflicts by the time it was approved.

## Changelog
:cl:
add: Added Enginseer Robes to the Robotics vendor contraband list.
/:cl:

### CODE REVIEW

File: code/modules/vending/wardrobes.dm [hunk 1]
```diff
@@ -161,6 +161,7 @@
 					/obj/item/clothing/mask/bandana/skull = 2
 					)
 	contraband = list(/obj/item/clothing/suit/hooded/techpriest = 2,
+					/obj/item/clothing/suit/hooded/enginseer = 2,
```
Comment (Shadowtail117) [inline code/modules/vending/wardrobes.dm:164]:
```suggestion
					/obj/item/clothing/suit/hooded/enginseer = 2, // WS edit: enginsineer robes
```
Margularization be damned.

**Review by Shadowtail117 — CHANGES_REQUESTED** [review]

**Comment by Shadowtail117** [comment]
The changelog is also a bit verbose, it's implied if you're adding an item that you're also adding it to the code and sprite files. I recommend something like:

> :cl:
> add: Added Enginsineer robes to the robotics clothing vendor contraband list
> /:cl:

**Comment by DerFlammenwerfer** [comment]
> The changelog is also a bit verbose, it's implied if you're adding an item that you're also adding it to the code and sprite files. I recommend something like:
> 
> > 🆑
> > add: Added Enginsineer robes to the robotics clothing vendor contraband list
> > /🆑

Alright, done.

**Review by Shadowtail117 — APPROVED** [review]
Tested locally, works like a charm.

**Pull request merged**